### ISSUE TITLE
Fix typo in ArgumentMatcher documentation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dlsgusrn7577/mockito#356]
```
s/decamlizes/decamlizes. You know, I had to google search for camlize before I 
realized what you meant? :)
```

Original issue reported on code.google.com by `konigsb...@google.com` on 25 Jul 2012 at 5:38

Attachments:
- [ArgumentMatcher.java.patch](https://storage.googleapis.com/google-code-attachments/mockito/issue-356/comment-0/ArgumentMatcher.java.patch)

**Comment by GoogleCodeExporter** [comment]
```
Hi, Thanx for the patch :)
Fixed in revision bc60cb3d5637
```

Original comment by `brice.du...@gmail.com` on 1 Aug 2012 at 3:39
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `szcze...@gmail.com` on 6 Oct 2012 at 7:38
- Added labels: **Milestone-Release1.9.5**
